assets/img/segments-locked.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="142" height="18" viewBox="0 0 142 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This svg is not used anymore right? We should delete this if it's not used anywhere.